DinamicNavMenu/about.html

```diff
@@ -0,0 +1,16 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
+</head>
+<body>
+    <h1 id="h1">About</h1>
+    <a id="firstA" href="./home.html">Home</a>
+    <a id="secondA" href="./about.html">About</a>
+    <a id="thirdA" href="./contact.html">Contact</a>
```

Comment:
os id's podiam ser mais explícitos
```suggestion
    <a id="homeLink" href="./home.html">Home</a>
    <a id="aboutLink" href="./about.html">About</a>
    <a id="contactLink" href="./contact.html">Contact</a>
```